### DISCUSSION TITLE
Convert 16-bit grayscale BMP to 16-bit grayscale PNG?

### DISCUSSION THREAD

**Discussion by metasprite** [ImageMagick/ImageMagick#3525] [Help]
I have a 512x512, 16-bit grayscale BMP. It is from an old game's editing tools, and I want to export it. The only option available for export is 16-bit grayscale BMP. This screenshot of the game's editor shows what the image should look like, and that it is indeed 16-bit grayscale:

![image](https://user-images.githubusercontent.com/25594046/114256493-2ffdf180-997f-11eb-93f1-7e3523a8596e.png)

However, both the Windows preview and ImageMagick mistake it for an sRGB BMP image, thinking it looks like this:

![image](https://user-images.githubusercontent.com/25594046/114256509-50c64700-997f-11eb-8278-901cc2b29946.png)

I'm guessing it's because the format is unspecified in the BMP header. There's _absolutely nothing_ I can do about that, it's a heightmap exported from old proprietary software. I really need to convert that 16-bit grayscale data to a 16-bit grayscale PNG but `convert` isn't cooperating:

![image](https://user-images.githubusercontent.com/25594046/114256610-f5e11f80-997f-11eb-9270-7d83b77d8e18.png)

It produces this incorrect result:

![image](https://user-images.githubusercontent.com/25594046/114256614-01cce180-9980-11eb-8c6b-06d0a274db77.png)

Surely there must be some way to force `convert` to treat the input as 16-bit grayscale instead of sRGB?

Here's the 512kb grayscale bitmap file, since GitHub won't let me upload a bmp: https://www.mediafire.com/file/ede91u6lewvxv3d/Heightmap.bmp/file

**Comment by snibgo** [comment]
To download from mediafire, the website needs my consent to steal my personal data, which I won't give.

Please use another website. You might put your BMP into a zip file first, to prevent a website messing with it.
